.chloggen/ottl-improve-parsing-errors.yaml

```diff
@@ -0,0 +1,22 @@
+# Use this changelog template to create an entry for release notes.
+# If your change doesn't affect end users, such as a test fix or a tooling change,
+# you should instead start your pull request title with [chore] or use the "Skip Changelog" label.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
```

Comment:
What are our stability guarantees for error messages? Does changing an error message count as a breaking change? I don't feel like the error messages that have changed here are anything we want users parsing, but maybe I'm ignoring some valid use cases.

Reply:
Because there is a chance that users could be relying on them for monitoring there is an argument that it is a breaking change.

Reply:
What makes me resistant to making stability guarantees about the content of the error messages is that they are unstructured. Regardless, you're right someone may depend on them, so I'm fine playing it safe and making this a breaking change.